utils/post.js

```diff
@@ -333,13 +335,14 @@ const deleteExpiredPostFromFeed = (item, feed_id) => {
   }
 };
 
-const modifyNewItemAnonymity = (newItem, isBlurredPost = false) => {
+const modifyNewItemAnonymity = (newItem, isBlurredPost = false, anonymUserId) => {
   if (newItem.anonimity) {
+    const isOwnAnonPost = newItem?.actor?.id === anonymUserId;
+    newItem.isBlurredPost = isOwnAnonPost ? false : isBlurredPost;
     newItem.actor = {};
     newItem.to = [];
     newItem.origin = null;
     newItem.object = '';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, in the function `modifyNewItemAnonymity`, the new parameter `anonymUserId` is added after a parameter with a default value (`isBlurredPost = false`). This could lead to confusion and potential errors. Parameters with default values should be placed at the end of the parameter list.
```diff
- const modifyNewItemAnonymity = (newItem, isBlurredPost = false, anonymUserId) => {
+ const modifyNewItemAnonymity = (newItem, anonymUserId, isBlurredPost = false) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->